src/button-group.test.visuals.ts

```diff
@@ -8,7 +8,7 @@ for (const story of stories['Button Group']) {
   test.describe(story.id, () => {
     for (const theme of story.themes) {
       test.describe(theme, () => {
-        test('hover', async ({ page }, test) => {
+        test(':hover', async ({ page }, test) => {
```

Comment:
Switched to `':hover'` and `':active'` throughout to accommodate [this](https://github.com/CrowdStrike/glide-core/pull/760/files#diff-6df8d29db5f0ac4adb65f0aa3c07a0f17737ea29e945e3e6db17741ad45e1e7bR78) case where `'hover'` and `'press'` didn't make sense.